src/Magnum/DartIntegration/Test/DartSkeletonTest.cpp

```diff
@@ -0,0 +1,267 @@
+    dart::dynamics::SkeletonPtr pendulum = dart::dynamics::Skeleton::create("pendulum");
+
+    // Add each body to the last BodyNode in the pendulum
+    dart::dynamics::BodyNode* bn = makeRootBody(pendulum, "body1");
+    bn = addBody(pendulum, bn, "body2");
+    bn = addBody(pendulum, bn, "body3");
+    bn = addBody(pendulum, bn, "body4");
+    bn = addBody(pendulum, bn, "body5");
+
+    // Create a world and add the pendulum to the world
+    dart::simulation::WorldPtr world(new dart::simulation::World);
+    world->addSkeleton(pendulum);
+
+    Scene3D scene;
+    Object3D* obj = new Object3D{&scene};
+
+    DartSkeleton* skel = new DartSkeleton{obj, pendulum};
+
+    // Set the initial joint positions so that the pendulum
+    // starts to swing right away
+    pendulum->getDof(1)->setPosition(120. * M_PI / 180.0);
+    pendulum->getDof(2)->setPosition(20. * M_PI / 180.0);
+    pendulum->getDof(3)->setPosition(-50. * M_PI / 180.0);
+
+    world->step();
+    skel->updateObjects();
+
+    auto objects = skel->objects();
+
+    auto trans = bn->getShapeNodesWith<dart::dynamics::VisualAspect>().back()->getTransform();
+
+    auto R = Eigen::AngleAxisd(trans.linear());
+    auto axis = R.axis();
+    auto T = trans.translation();
+
+    // Convert it to axis-angle representation
+    Math::Vector3<Float> t = {Float(T[0]), Float(T[1]), Float(T[2])};
+    Math::Vector3<Float> u = {Float(axis(0)), Float(axis(1)), Float(axis(2))};
+    Float theta = Float(R.angle());
+
+    Math::Matrix4<Float> transformation;
+    transformation = Math::Matrix4<Float>::translation(t) * Math::Matrix4<Float>::rotation(Math::Rad<Float>(theta), u);
+
+    CORRADE_VERIFY(objects.size() == 15);
+    CORRADE_VERIFY(objects.back()->object().absoluteTransformationMatrix() == transformation);
+}
+
+void DartSkeletonTest::test_urdf() {
+    dart::utils::DartLoader loader;
+    std::string filename = std::string(DARTINTEGRATION_TEST_DIR) + std::string("/urdf/test.urdf");
+    auto tmp_skel = loader.parseSkeleton(filename);
+    CORRADE_VERIFY(tmp_skel);
+
+    dart::simulation::WorldPtr world(new dart::simulation::World);
+    world->addSkeleton(tmp_skel);
+
+    Scene3D scene;
+    Object3D* obj = new Object3D{&scene};
+
+    DartSkeleton* skel = new DartSkeleton{obj, tmp_skel};
+
+    world->step();
+    skel->updateObjects();
+    size_t pts[] = {1524, 1682, 2554, 1514, 2442, 706, 3872};
+
+    size_t j = 0;
+    for (size_t i = 0; i < skel->objects().size(); i++) {
+        if (!skel->objects()[i]->shapeNode())
+            continue;
+        auto shape = skel->objects()[i]->shapeNode()->getShape();
+        CORRADE_VERIFY(shape->getType() == "MeshShape");
+        auto mydata = convertShapeNode(skel->objects()[i]->shapeNode());
+        CORRADE_VERIFY(mydata);
+        CORRADE_VERIFY((*mydata)._meshData->positions(0).size() == pts[j]);
+        Vector3 color{0.6f, 0.6f, 0.6f};
+        CORRADE_VERIFY((*mydata)._matData->diffuseColor() == color);
```

Comment:
Using `CORRADE_COMPARE((*mydata)._matData->diffuseColor(), color)` instead of `CORRADE_VERIFY()` improves the diagnostic output better in case of a failure (i.e., it prints both values instead of just saying that comparison failed). That is great to have when debugging test failures on the CI, for example. Would be great if you could use it in all cases where it's comparing more than just a boolean :)